packages/tools/devtools/devtools-view/src/components/ContainerHistoryLog.tsx

```diff
@@ -76,19 +82,25 @@ export function ContainerHistoryLog(props: ContainerHistoryLogProps): React.Reac
 
 	const getBackgroundColorForState = (state: string): string => {
 		switch (state) {
-			case "attached":
-				return tokens.colorPaletteRoyalBlueBackground2; // blue
-			case "closed":
-				return tokens.colorPaletteRedBorder1; // red
-			case "connected":
-				return tokens.colorPaletteGreenBackground2; // green
-			case "disconnected":
-				return tokens.colorPaletteDarkOrangeBorderActive; // orange
-			case "disposed":
-				return tokens.colorPaletteDarkRedBackground2; // dark red
-			default:
+			case "attached": {
+				return tokens.colorPaletteRoyalBlueBackground2;
+			} // blue
```

Comment:
The comments here appear to have been moved by eslint. Would you mind moving them back to the return statements?

Reply:
This is why pre-pending comments instead of putting them at the end of lines is better. Fixed in future iteration.